### ISSUE TITLE
Clarify proper usage with `cmdheight=0`

### ISSUE THREAD

**Issue by Mateiadrielrafael** [jbyuki/venn.nvim#28]
It looks like the hydra suggested keybinds print `:VBox` to the cmd line even when `silent = true`, which make the statusline flicker when `cmdheight=0`. I cannot figure out how to use `:silent` with a visual mode command, so it would be nice if the readme suggested a way to fix issues like these.

**Comment by jbyuki** [comment]
This is more related to how hydra handles keybindings. Although I know that the maintenainer is currently not very active so it would be hard to get answers there. How about setting the keybinding manually?
```lua
vim.keymap.set('v', '<leader>g', ':VBox<cr>', {silent = true})
```
or using vimscript
```lua
vnoremap <silent><leader>g :VBox<CR>
```

**Comment by Mateiadrielrafael** [comment]
@jbyuki You are right, setting the keymap using `vim.keymap.set` does not make the cmdline flicker, although I would expect setting `silent = true` to have the same effect in hydra (it does not).

**Comment by jbyuki** [comment]
I've seen a new hydra.nvim fork scrolling in my feed: https://github.com/nvimtools/hydra.nvim . Maybe they could help you there on that subject as I don't use hydra myself.

**Comment by Mateiadrielrafael** [comment]
Never mind, the issue still occurs with the vim keybind:
![neovim-statusline-flicker](https://github.com/jbyuki/venn.nvim/assets/39400800/41743995-09f3-4803-abdf-aa833f5f69e6)


@jbyuki

**Comment by jbyuki** [comment]
Related to the status line plugin? It doesn't flicker for me.
